README.md

```diff
@@ -151,7 +151,7 @@ petitionsclerk1@example.com – petitionsclerk5@example.com
 docketclerk1@example.com – docketclerk5@example.com
 respondent1@example.com – respondent10@example.com
 practitioner1@example.com – practitioner10@example.com
-seniorattorney1@example.com – seniorattorney5@example.com
+adc1@example.com – adc5@example.com
```

Comment:
`test-all.sh` routinely fails for me — it dies with this:

```
 91  278M   91  254M    0     0  1720k      0  0:02:45  0:02:31  0:00:14 1219kWaiting for http://localhost:9200/ to be hosted...

 91  278M   91  255M    0     0  1717k      0  0:02:45  0:02:32  0:00:13 1200kWaiting for http://localhost:3000/api/swagger to be hosted...

 92  278M   92  257M    0     0  1716k      0  0:02:45  0:02:33  0:00:12 1264kWaiting for http://localhost:9200/ to be hosted...

 92  278M   92  258M    0     0  1714k      0  0:02:46  0:02:34  0:00:12 1296kWaiting for http://localhost:3000/api/swagger to be hosted...

 93  278M   93  259M    0     0  1713k      0  0:02:46  0:02:35  0:00:11 1391kWaiting for http://localhost:9200/ to be hosted...

 94  278M   94  261M    0     0  1713k      0  0:02:46  0:02:36  0:00:10 1513kWaiting for http://localhost:3000/api/swagger to be hosted...

 94  278M   94  263M    0     0  1713k      0  0:02:46  0:02:37  0:00:09 1567kWaiting for http://localhost:9200/ to be hosted...

 94  278M   94  264M    0     0  1708k      0  0:02:46  0:02:38  0:00:08 1449kWaiting for http://localhost:3000/api/swagger to be hosted...

:(  giving up.
```

Please resolve this.

Reply:
We've added 2 additional cache resets to this script. Joe and I are both able to run this script successfully.

Reply:
That did it! Thank you. :)